api/graphql/tests/test_reservation_units.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+import snapshottest
+from assertpy import assert_that
+from graphene_django.utils import GraphQLTestCase
+from rest_framework.test import APIClient
+
+from reservation_units.tests.factories import ReservationUnitFactory
+
+
+class ReservationUnitCapacityTestCase(GraphQLTestCase, snapshottest.TestCase):
```

Comment:
Test class naming could be like `ReservationUnitTestCase`

Reply:
Ooops, copy pasta.